variables.tf

```diff
@@ -510,6 +510,11 @@ variable "redis_subnet_tag_filter" {
   default     = {}
 }
 
+variable "redis_subnet_cidr_block_filter" {
```

Comment:
Please sepcify the type of variable explicitely

Reply:
Done with [6232c30](https://github.com/Flaconi/terraform-aws-microservice/pull/62/commits/6232c305a6045017894e7f7d13a9f0bd647577d7).